docs/installation/addnode_via_penetrator_cn.md

```diff
@@ -0,0 +1,82 @@
+# 使用penetrator添加边缘节点
+
+## 安装节点
+
+### 通过master节点安装节点
+
```

Comment:
More detailed information is expected, such as, working principle, preconditions, how to observe the operation